dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.0.json

```diff
@@ -957,7 +968,7 @@
     {
      "type": "html",
      "name": "thank_you_header",
-     "html": "<h2> Thank You!</h2>\n<p>Thanks for your time and commitment to teaching computer science</p>"
+     "html": "<img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"https://www.jotform.com/uploads/bakerfranke/form_files/codeorg_heart_you.5cc89a7c7f2f20.88587690.png\" />\n<p>Thanks from Code.org for your time, commitment and passion for teaching {workshop_course} this year.</p>\n<p><strong>You are making a difference for your students.</strong> We're looking forward to what's next. Onward!</p>"
```

Comment:
As discussed in a recent thread, this file should be hosted by us, if we use it.